app/src/routing.ts

```diff
@@ -57,9 +57,3 @@ export const useIsCreateRoute = () => {
   let routematch = useRouteMatch(createRoute);
   return routematch !== null;
 };
-export const useShouldHaveBlackHeaderBackground = () => {
```

Comment:
Hmm, hvorfor blir denne slettet? Ser du ikke har merget/rebaset inn endringer fra master. Tror det er kjekt å gå gjort før det merges

Reply:
Uh prøvde å flytte den til komponenten der den brukes, men skal merge!

Reply:
Trekker det tilbake, eg var up to date med master. Ser du på en gammel commit? Eller er det du som ikkje har merga til mester?

Reply:
ser ut til at den ble flyttet inn i Header.tsx en eller annen gang. my bad